src/test/fuzz/txrequest.cpp

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <primitives/transaction.h>
+#include <random.h>
+#include <txrequest.h>
+#include <test/fuzz/fuzz.h>
+#include <crypto/common.h>
+#include <crypto/siphash.h>
+
+#include <bitset>
+#include <cstdint>
+#include <queue>
+#include <vector>
+
+namespace {
+
+constexpr int MAX_TXHASHES = 16;
+constexpr int MAX_PEERS = 16;
+
+//! Randomly generated GenTxids used in this test (length is MAX_TX).
```

Comment:
8d0dd46b4fcdf4c664747e638eda046f8d51079c
```suggestion
//! Randomly generated GenTxids used in this test (length is MAX_TXHASHES).
```

Reply:
Done.